### PR TITLE
Update cases to us 0.10 format instead of 0.9 format

### LINKED ISSUE #11: Update from Spec version 9 -> 10
- [ ] keys have been changed from case-insensitive with `-` to all lower case with `_` (`Link-Location` -> `link_location`)
- [ ] other things?

### PR DESCRIPTION
**Pull request by dcbaker**
This changes the keys from `Proper-Case` -> `proper_case`.

I've sent this as two commits despite the fact that it's one logical change because the first commit is largely machine generated (with manual touch ups), while the second is human generated. Since the script used for the conversion is in the commit message for the first patch, this seems like a good case for a merge, despite the fact that I normally dislike merges.

Fixes #11

### CODE REVIEW

File: tests/cases/lib/cps/cps-path-set.cps [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+            "location": "@prefix@/lib/libfoo.a"
         }
     },
-    "Default-Components": ["default"]
-}
-
+    "default_components": [
+        "default"
+    ]
+}
```
Comment (tylerjw) [inline tests/cases/lib/cps/cps-path-set.cps:20]: newline end of file?

File: tests/cases/lib/cps/minimal.cps [hunk 2]
```diff
@@ -1,22 +1,51 @@
...
+            },
+            "location": "fake"
         }
     },
-    "Default-Components": ["sample1"]
-}
+    "default_components": [
+        "sample1"
+    ]
+}
```
Comment (tylerjw) [inline tests/cases/lib/cps/minimal.cps:51]: here too

**Review by tylerjw — APPROVED** [review]
Looks good

File: tests/cases/lib/cps/cps-path-set.cps (hunk 1 above)
Comment (dcbaker) [reply tests/cases/lib/cps/cps-path-set.cps]: Weird that there were 3 cps files that didn't have final newlines, but the rest did. I've updated those

**Review by tylerjw — APPROVED** [review]

**Pull request merged**